src/pages/Product/ProductList.js

```diff
@@ -0,0 +1,72 @@
+import React, { useState } from 'react';
+const colorBtn = [
+  {
+    id: 1,
+    btnColor: 'red',
+    btnName: 'red',
+  },
+  {
+    id: 2,
+    btnColor: 'blue',
+    btnName: 'blue',
+  },
+  {
+    id: 3,
+    btnColor: 'orange',
+    btnName: 'orange',
+  },
+  {
+    id: 4,
+    btnColor: 'khaki',
+    btnName: 'khaki',
+  },
+  {
+    id: 5,
+    btnColor: 'black',
+    btnName: 'black',
+  },
+  {
+    id: 6,
+    btnColor: 'navy',
+    btnName: 'navy',
+  },
+];
+
+const ProductList = ({ product }) => {
+  const { id, src, src2, name, price, color } = product;
+  const [imgHover, setImgHover] = useState(false);
+
+  return (
+    <div className="productOne" productColor={color} key={id}>
+      <img
+        src={imgHover ? src : src2}
+        className="productImg"
+        alt="product"
+        onMouseOver={() => setImgHover(true)}
+        onMouseOut={() => setImgHover(false)}
+      />
+      <div className="productColor">
+        {colorBtn.map(btn => {
+          const { id, btnColor } = btn;
```

Comment:
데이터양이 많지 않다면 파라미터 자리에서 구조분해 해주는 것도 하나의 방법입니다!
```suggestion
        {colorBtn.map(({id,btnColor})=> {
```